### DISCUSSION TITLE
Budgie 10.6.1

### DISCUSSION THREAD

**Discussion by JoshStrobl** [BuddiesOfBudgie/budgie-desktop#98] [Announcements]
Budgie 10.6.1 is the first minor release that introduces fixes for bugs in the 10.6 series of our Budgie Desktop environment.

Release notes are available [here](https://blog.buddiesofbudgie.org/budgie-10-6-1-released/).

<hr /><em>This discussion was created from the release <a href='https://github.com/BuddiesOfBudgie/budgie-desktop/releases/tag/v10.6.1'>Budgie 10.6.1</a>.</em>

**Comment by serebit** [comment]
Cool release, could use more birds though
